lib/cylc/config.py

```diff
@@ -535,13 +558,16 @@ def __init__(self, suite, fpath, template_vars=[], template_vars_file=None,
 
         # Warn or abort (if --strict) if naked dummy tasks (no runtime
         # section) are found in graph or queue config.
-        if len( self.naked_dummy_tasks ) > 0:
+        if len(self.naked_dummy_tasks) > 0:
             if self.strict or flags.verbose:
-                print >> sys.stderr, 'WARNING: naked dummy tasks detected (no entry under [runtime]):'
+                print >> sys.stderr, (
+                    'WARNING: naked dummy tasks detected (no entry under ' +
```

Comment:
The `+` at EOL isn't necessary here (and there are a bunch of these) ... it doesn't really matter, however.